snet_cli/session.py

```diff
@@ -16,6 +16,8 @@ def from_config(config):
         identity = SessionIdentity(
             eth_rpc_endpoint=config.get("identity.{}".format(config.get("session", "identity_name")),
                                         "eth_rpc_endpoint", fallback=None),
+            ipfs_endpoint=config.get("identity.{}".format(config.get("session", "identity_name")),
```

Comment:
wait, how is `ipfs_endpoint` part of identity at all?

Reply:
Removed